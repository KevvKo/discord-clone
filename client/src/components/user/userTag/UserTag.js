function UserTag() {

    const userName = "unosxmopsus";
    const userTag = "#3432";

    return (
    <div className="user-tag flex-fill">
        <div>{userName}</div>
        <div>{userTag}</div>
    </div>
    );
}

export default UserTag;
