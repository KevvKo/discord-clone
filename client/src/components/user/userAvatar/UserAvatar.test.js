import { render, screen } from '@testing-library/react';
import UserTag from './UserAvatar';

test('renders learn react link', () => {
  render(<UserTag />);
  const linkElement = screen.getByText(/ xyz /i);
  expect(linkElement).toBeInTheDocument();
});
