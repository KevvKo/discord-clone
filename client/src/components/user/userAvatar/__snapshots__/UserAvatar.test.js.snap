// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserAvatar should render with given state from redux store 1`] = `
<div
  className="user-avatar mr-2"
  data-testid="user-avatar"
>
  <i
    className="bi bi-person-circle"
    style={
      Object {
        "fontSize": "1.666rem",
      }
    }
  />
  <div
    className="active-user d-inline-block"
  />
</div>
`;
